windows/CodePushResumeListener.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ReactNative.Bridge;
+using System;
+
+namespace CodePush.ReactNative
+{
+    internal class CodePushResumeListener : ILifecycleEventListener
```

Comment:
I would vote to rename this to "LifecycleEventListener". We listen to suspend as well, so it isn't just a resume listener, and the CodePush prefix doesn't add any value since the namespace already scopes it down pretty specifically.